### DISCUSSION TITLE
Couple of questions

### DISCUSSION THREAD

**Discussion by moritzbeck13** [seerge/g-helper#487] [General]
What exactly do the CPU boost options under the custom fan curve profiles mean/represent?

Can I also show fan % instead of RPM in the custom fan curve profiles?

Is there a `restore defaults` function for PPTs?

What is the `Asus System Control Interface` needed for? Never had Armoury Crate or MyAsus on this install and all seems to work fine. I also can't find this process running in Task Manager.

**Comment by seerge** [comment]
> What exactly do the CPU boost options under the custom fan curve profiles mean/represent?

Shortcut to same setting in your windows power plan.

> Can I also show fan % instead of RPM in the custom fan curve profiles?

On charts - you can't, on main app window - clicking toggles % / rpm

> 
> Is there a `restore defaults` function for PPTs?

Unless "Apply PPT" is checked - default ones are used (i.e. sliders are ignored)

> 
> What is the `Asus System Control Interface` needed for? Never had Armoury Crate or MyAsus on this install and all seems to work fine. I also can't find this process running in Task Manager.

It's driver, not a process. It's being installed by windows update. That's why you have it.

**Reply by moritzbeck13** [reply]
Thanks a ton!

> Shortcut to same setting in your windows power plan.

Sorry, but I don't see what you mean. In my windows power plan I don't have any of these options.

> On charts - you can't, on main app window - clicking toggles % / rpm

Can I open a feature request for this or is there any reason behind it?

**Reply by seerge** [reply]
> Sorry, but I don't see what you mean. In my windows power plan I don't have any of these options.

Because you need to do some regedit before it appears (use google)

> Can I open a feature request for this or is there any reason behind it?

Reason behind is that % (the way it's done by asus) is confusing. As 1% for asus hardware immediately means 1800RPM, and 100% means whatever maximum is (~6000RPM).

**Reply by moritzbeck13** [reply]
> Because you need to do some regedit before it appears (use google)
> 
> >

Got it, thanks! 👍



> Reason behind is that % (the way it's done by asus) is confusing. As 1% for asus hardware immediately means 1800RPM, and 100% means whatever maximum is (~6000RPM).

So are the RPM displayed actually reported by a fan speed sensor or is it calculated based on %? Because is seems way to stable for me to be an actual measured value.

**Reply by seerge** [reply]
RPMs are actually real, % in main window are calculated as % of total known max RPM for your fan (app auto-calibrates it if sees higher than pre-set RPMs). It's just a very "approximate" sensor there. It rounds to 100s

**Reply by moritzbeck13** [reply]
So wouldn't it make sense to set it based on % but report based on RPM? Because I'm guessing on a hardware level setting is done as a relative value.

The reason for me asking is that, on a high resolution display, choosing specific RPMs kinda is a pita. Could we maybe get snapping to nearest 100 or entering a value manually? Otherwise there is no way to reliably choose a precise setpoint, except for editing the config manually maybe.

**Reply by seerge** [reply]
@moritzbeck01 fans are not precise anyway, bios will fluctuate them anyway. Just approximate setting is good enough.

**Comment by kuonji-arisu** [comment]
You asked the question I wanted to ask XD.

I don’t quite understand the developer’s answer, because I didn’t find the corresponding shortcut option in the power plan. I haven’t learned .net, but I used rider to look at the GHelper code, and I found something.

This option called CPU Boost actually calls the external method PowerWriteACValueIndex. It changes the PERFBOOSTMODE value in the GUID_PROCESSOR_SETTINGS_SUBGROUP subgroup (control configuration of processor power management) in the current power plan, which has alternative values from 0-5, corresponding to disabled and Efficient aggressive in the options. 

But I don’t understand if this option is unique to GHelper, or a port of a feature that comes with Asus ac.

**Reply by seerge** [reply]
@kuonji-arisu you may want to check this https://www.reddit.com/r/ZephyrusG14/comments/gho535/important_update_to_properly_disable_boosting/ 

it unhides the setting in windows power plan, that app offers you shortcut to. That's it.

**Reply by kuonji-arisu** [reply]
> @kuonji-arisu you may want to check this https://www.reddit.com/r/ZephyrusG14/comments/gho535/important_update_to_properly_disable_boosting/
> 
> it unhides the setting in windows power plan, that app offers you shortcut to. That's it.

Thanks! Does Armoury Crate automatically adjust this setting when switching modes?

**Reply by seerge** [reply]
No
